classes/form/Form.inc.php

```diff
@@ -303,8 +303,10 @@ function validate($callHooks = true) {
 	/**
 	 * Execute the form's action.
 	 * (Note that it is assumed that the form has already been validated.)
+	 * @param mixed $functionArgs,... Arguments from the caller to be passed to the hook consumer
+	 * @return mixed Result from the consumer to be passed to the caller.  Send a true-ish result if you want the caller to do something with the return value.
 	 */
-	function execute() {
+	function execute(...$functionArgs) {
```

Comment:
Points for the splat operator! But I'd suggest also using that in the `HookRegistry::call` function, for consistency.

Reply:
Now variable is used for consistency.